.github/workflows/project-automation.yml

```diff
@@ -0,0 +1,11 @@
+name: project-management
```

Comment:
I'm waiting for our PM's confirmation if we want this repo to be included into project automation. We've so far been using it only for Conduit itself and the built-in connectors.

Reply:
Removed